ibm/ibm-how-to/launch-ubuntu-instances.rst

```diff
@@ -0,0 +1,153 @@
+Launch a Virtual Server Instance on IBM Cloud
+=============================================
+
+This tutorial guides you through the process of launching a Virtual Server Instance (VSI) on IBM Cloud using Ubuntu. Ensure you have an active IBM Cloud account before starting.
+
+Via Web Console
+---------------
+
+#. First, on the IBM Cloud Web Console, navigate to the Virtual Server creation form. To do this, navigate to     
+   :guilabel:`VPC Infrastructure` > :guilabel:`Compute`> :guilabel:`Virtual server instances`>, 
+   and then click on the blue :guilabel:`Create +`.  
+#. Enter a unique name for your VSI in the `Name` field.
+#. Under the image section, press on :guilabel:`Change image` and under the :guilabel:`Stock Images` tab, search for
+   "Ubuntu" in the search bar and select the Ubuntu OS you'd like to use. For the most recent OS, select :code:`ibm-ubuntu-24-04-minimal-amd64-1`
+#. Choose the desired profile for your VSI under the `Profile` section.
+#. Select an existing ssh key or create a new one by pressing on :guilabel:`Add SSH key` and following the instructions.
+#. Customize any other settings as needed.
+#. Press on the blue :guilabel:`Create virtual server` button to launch your VSI.
+
+Via IBM Cloud CLI
+-----------------
+
+**Prerequisites**
+
+- IBM Cloud account
+- IBM Cloud CLI installed on Ubuntu
+
+
+**Step 1: Log in to IBM Cloud**
+
+1. After installing the CLI, log in to your IBM Cloud account using the command:
+
+.. code-block:: bash
+
+   ibmcloud login
+
+2. If you have a federated ID, use:
+
+.. code-block:: bash
+
+   ibmcloud login --sso
+
+**Step 2: Target a Resource Group**
+
+1. List available resource groups:
+
+.. code-block:: bash
+
+   ibmcloud resource groups
+
+2. Target the desired resource group:
+
+.. code-block:: bash
+
+   ibmcloud target -g <Resource_Group_Name>
+
+**Step 3: Create SSH Key**
+
+1. Generate an SSH key for VSI access if you don't have one:
+
+.. code-block:: bash
+
+   ssh-keygen -t rsa -b 2048
+
+2. Add the SSH key to IBM Cloud:
+
+.. code-block:: bash
+
+   ibmcloud is key-create MyKey --public-key ~/.ssh/id_rsa.pub
```

Comment:
This command doesn't seem to work.

I used `ibmcloud is key-create "uhryniuk-testing" --public-key ~/.ssh/id_rsa.pub` and got the result,
```
dylan_uhryniuk@cloudshell:~/.ssh$ ibmcloud is key-create MyKey --public-key ~/.ssh/id_rsa.pub
Incorrect Usage: flag provided but not defined: -public-key

NAME:
    key-create - Import an RSA public key
  
USAGE:
    ibmcloud is key-create KEY_NAME (KEY | @KEY_FILE) [--key-type rsa | ed25519] [--resource-group-id RESOURCE_GROUP_ID | --reso
urce-group-name RESOURCE_GROUP_NAME] [--output JSON] [-q, --quiet]
    KEY_NAME: ID or name of the key.
    KEY:      key|@key-file. The public SSH key to import into the system.

EXAMPLE:
    ibmcloud is key-create my-key "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDL9osaBrUD8uCBzIJo5YBvX8wtGrE+kcC7YZtID/nNYrjeCB26eFASH
ia5tmqmuCo434UygGSd5qj3t/3v/a7NZoMr/0+qspQF+dUVIl+xIsKTWYQ+gtYbJlvW+FIlNTOA4vbOXLg+nGGUCoaV79azmny4mYJbbo15i+Q3CI+w9bwOAwzqeGKae
Ojpo5hdDcFW0QLDxKmQHKMLX8slsx3kB9I5wPe8C/ZBBDBBkZKK2y3RJBjaKxi0beFueo6ngUKOLooReefiBGpdoOJIi6Gf7vRduoBTmbyVvSv08wcrANtYSzGwDpqrE
shEafv8bKo42MYHsPT2OwAbsFyqWQj5 test@example"
    ibmcloud is key-create my-key @/tmp/my_id_rsa.pub
    ibmcloud is key-create my-key @/tmp/my_id_rsa.pub --output JSON
    ibmcloud is key-create my-key "ssh-ed25519 AAAAC3NzaC1lZDI1NTE6AAAAID/R2T8h6CPvZr/InxpBrxh8bmG2RTyB8vzUTvOtQhaJ test@example
.com" --key-type ed25519
    ibmcloud is key-create my-key @/tmp/my_id_ed25519.pub --key-type ed25519
    ibmcloud is key-create my-key @/tmp/my_id_ed25519.pub --key-type ed25519 --output JSON  OPTIONS:    --key-type value             The crypto-system used by this key. Default is rsa. One of: rsa, ed25519.
    --resource-group-id value    ID of the resource group. This ID is mutually exclusive with --resource-group-name
    --resource-group-name value  Name of the resource group. This name is mutually exclusive with --resource-group-id
    --output value               Specify output format, only JSON is supported. One of: JSON.
    -q, --quiet                  Suppress verbose output

```